Fixes: #10065 fix mailserver popup showing after canceling it

cammellos opened the pull request:
Reopening this as it's still a valid issue, for example if the mailserver was initially online but later it went offline, the popup interferes with the user being able to change settings or ignore the warning.
 Original PR

https://github.com/status-im/status-react/pull/10342
### Summary

Fixes #10065 by setting a `dismissed` flag on Cancel, so the popup will not appear again.
Selecting a new mail server sets `dismissed` as false, so the error can appear again

#### Platforms

Tested Android simulator

### Steps to test

See issue above to see reproduction steps

jrainville committed:
fix: fix mailserver popup showing after canceling it

Fixes #10065
After clicking cancel, the popup is `dismissed`, so it will not appear again
Selecting a new mail server sets `dismissed` as false, so the error can appear again

jrainville committed:
fix: linting and variable name

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | cf61d034 | [#1](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-11495/1/) | 2020-12-01 09:03:23 | ~13 min | `android-e2e` | [:package:`apk`](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-201201-085022-cf61d0-pr-x86.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-prs.ams3.digitaloceanspaces.com%2FStatusIm-201201-085022-cf61d0-pr-x86.apk)|

flexsurfer commented:
i still think flow should be redesigned from scratch, current UI flow is broken, but even if we want to solve this current case, the root of the problem is that we receive `mailserver/check-connection-timeout` from the timer, and if we'll wait some time with opened popup we'll have several popups anyway (this is not solved in this PR i believe) to solve it properly we need to stop the timer when we receive an error first time, or use timeout, also we need to clear timeout on cancel

cammellos commented:
> q: do we show same alert if its manual but not a custom mailserver ?

Yes, we show the same alert, which I believe is the right behavior (say the IP has been banned by your ISP, you want to have the same behavior).

> the problem is that auto selection and manuall selections are mixed, and also UI flow for custom mailservers is different, probably as a proper solution we need to stop async-periodic when we receive error first time if mode is manual

We could, but that's a critical path of the code and I'd rather not touch it, before we had the timer we spend ages trying to fix issues with the mailserver not reconnecting anymore because of state management, so having a watchdog always on though is not an elegant solution, it makes me sleep safer at night.

I agree with you that there might be better solutions, but currently there's little appetite to change this code and this to me feels like an ok solution, and it addresses the issue that user have reported (just a couple of days ago), so to me this is good enough, the next natural step would be to move this logic to status-go, so we can simplify the UI code and it's easier to test.

What do you think?

status-im-auto commented:
## 97% of end-end tests have passed
```
Total executed tests: 91
Failed tests: 3
Passed tests: 88
```
```
IDs of failed tests: 6237, 6263, 6251 
```
<h3>Failed tests (3)</h3><details><summary>Click to expand</summary><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001104">test_fetching_balance_after_offline</a>, id: 6237 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on StatusInBackgroundButton</div><div>Device 1: Waiting 300 seconds for STT to update</div></blockquote></p><code> Device 1: Balance is not changed during 300 seconds!</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6eea4601032449be9b928d8db889fc59?auth=b9399c3b412d9b915ea8889930d64a16">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/6eea4601032449be9b928d8db889fc59/0075screenshot.png?auth=b9399c3b412d9b915ea8889930d64a16">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001058">test_request_and_receive_stt_in_1_1_chat_offline</a>, id: 6263 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on SignAndSend</div><div>Device 2: Tap on NextButton</div></blockquote></p><code> Device 2: 'NetworkFeeButton' is not found on the screen</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ef679f11a9e24c1595b5c6b33974ac01?auth=3eeb41027855e1b86e809a67df0fffba">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/ef679f11a9e24c1595b5c6b33974ac01/0030screenshot.png?auth=3eeb41027855e1b86e809a67df0fffba">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/67e0af12424749ef80b50335863e4303?auth=223b992cf7091e8885a45759e501c9be">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/67e0af12424749ef80b50335863e4303/0035screenshot.png?auth=223b992cf7091e8885a45759e501c9be">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001108">test_keycard_sign_message_from_daap</a>, id: 6251 </b></td></tr><tr><td><p><blockquote><div>Device 1: Type 'simpledapp.eth' to EnterUrlEditbox</div><div>Device 1: Tap 'Confirm' on native keyboard</div></blockquote></p><code> Device 1: 'AllowButton' is not found on the screen</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/34498745a506430e9cee8783f2caa5c7?auth=53f48e5b046a9a60364bd93c3f29981c">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/34498745a506430e9cee8783f2caa5c7/0036screenshot.png?auth=53f48e5b046a9a60364bd93c3f29981c">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (88)</h3><details><summary>Click to expand</summary><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001102">test_mobile_data_usage_popup_stop_syncing</a>, id: 6228 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/a883e0e5e4d845e89b4fc1cbeae8f3a8?auth=8174e66a3957cab5f66f560642e1dd1c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001034">test_back_forward_buttons_browsing_website</a>, id: 5321 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b3500115272d44a5b9209946913c4baa?auth=7686a2afd8bb68b93b5eaaeb301f557f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001074">test_wallet_set_up</a>, id: 5335 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/7440ba6e87ff4553b668576013d5e418?auth=0a2be8840ef89a1ef20c565ee94d2908">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001089">test_transaction_wrong_password_wallet</a>, id: 5408 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/527d296fb0a141e9bdd53c482ea4545b?auth=c998ed2b58fb55e52a55dda4b4b8e4d3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001106">test_add_and_delete_watch_only_account_to_multiaccount_instance</a>, id: 6244 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6fd46aff01954b6792798515c5ab593c?auth=dd67f958bb8d4a99b1d0f1a30e29ddba">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001052">test_install_pack_and_send_sticker</a>, id: 5782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/87b6ab9998eb45528458616b16d8cf67?auth=2975b990a437c52a50940bc02793837d">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/8e57b80de4534bbb80aa807d7b74c46d?auth=29652b78499e35a70a6379e4701ec7ef">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001071">test_user_can_switch_network</a>, id: 5299 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/91628ff3894b4da2beff3990f62c1f14?auth=91274967cfaf254fe6bb491aeb6a324b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001180">test_keycard_send_eth_to_ens</a>, id: 6295 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c9200dcf9287458bb3a828cb182af747?auth=36a6a525126c0de9e292bd1eb4d70c46">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001109">test_decline_transactions_in_1_1_chat_push_notification_changing_state</a>, id: 6265 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/8ac5bc33dc8540fb91b6d89fb031b371?auth=e2c13f48be03b91c3f85992e76526007">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/cf3a7cac26624351bc2835bb387613c4?auth=f6bb1a962fe7ccec0938f97b0d4abb59">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001047">test_start_chat_with_ens_mention_in_one_to_one</a>, id: 5403 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2297720738234a57af334c96bec36630?auth=3bcad4a6715ea2c9a65f7f01022f6dd5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001057">test_send_eth_in_1_1_chat</a>, id: 6253 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/42fcbdc7375d46f7bd47dd42ca24c368?auth=6d130089c556678cf074304d4f8bfb7a">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/f2ec991dadc643c185d73aa7a778cfe3?auth=773e29bf15eb909fe351044bbad6de76">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>12. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001025">test_send_eth_from_wallet_to_address</a>, id: 5308 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0e8bf420016545839a20f4e956a79188?auth=b768a7ea69a0eee97f93a1b3583022f2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>13. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001044">test_switch_users_and_add_new_account</a>, id: 5356 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b973a0e1066c4d13876665f26393cc17?auth=407d02295147145da92c6d5bc9d44a6d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>14. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001079">test_home_view</a>, id: 5379 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/97691b485f6d4eb1a93cb1a75a9a8769?auth=26f0800676c992a167feaff81e81f40b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>15. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001060">test_add_account_to_multiaccount_instance_private_key</a>, id: 6272 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d796f0fcfc8141b2935a520c903b51a3?auth=77ec87d905f547a3bb41202d8394f36a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>16. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001067">test_reactions_to_message_in_chats</a>, id: 6315 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/784d95ff7db04f83ac4d23574124544c?auth=294ba918b2ee2859d1466d1e7d255f5c">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/fda246d6903a42609b394e4cb9ae04df?auth=1d541629c7c447db671308000b6db725">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>17. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001055">test_add_account_to_multiaccount_instance_generate_new</a>, id: 6224 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/37e23fa11a064ac6b776d35e81c90d77?auth=bb7a8e1825e71d82bd362321dfd14b63">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>18. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001029">test_public_chat_messaging</a>, id: 5313 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/f2df1513ec1a4630ba3e3467bd3721c9?auth=e02ddaa9522782ea8f1739fcd34d0198">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/35e3793616e14bc39dfc773fe4fbd0e8?auth=2bc3a9e6d482b8e7b23d70fb33386d4b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>19. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001049">test_logcat_backup_recovery_phrase</a>, id: 5419 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6252c2815b154d1e8fcdad140978afcd?auth=891880e551b033772d5dec0ad3e58c8c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>20. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001101">test_open_blocked_secure_not_secure_sites</a>, id: 6210 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/a8c9831e3e9e415bb3bd798fdf7f81bb?auth=ddd897c0d3bf40c6fde71a0aea058feb">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>21. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001030">test_can_see_all_transactions_in_history</a>, id: 5314 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0871363050e840e8b710975faeda1fd7?auth=fba73154ffb032da2cd2bb338509e85e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>22. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001085">test_need_help_section</a>, id: 5391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2957ec735d654d3eb1d35f5ff9ff7ad5?auth=2bd69884358ccc2693c10ffb1be22dff">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>23. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001050">test_can_add_existing_ens</a>, id: 5502 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d6fb3eab955f4bbc8a817a6e670b3d15?auth=67619fed1b0ada7b5685fe2fc103fe97">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>24. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001059">test_add_account_to_multiaccount_instance_seed_phrase</a>, id: 6271 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2956d0d6700f49c599e9c469d6f96f4e?auth=3f1d15caf8d849a6429de0e8203e823b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>25. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001070">test_offline_add_new_group_chat_member</a>, id: 3998 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/da5dd76c960f4ed9b9b179a75c96adb9?auth=a84b507bebba2d7fc8d36006934d2975">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/7fe3ad9c236f4b38a29aeff2e3df8740?auth=e0e0b866cc0597f644c75bf26e4c5b66">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://saucelabs.com/jobs/98a4932281854ab38026a6d0c4d1c1c9?auth=73e3b1462c350558ae6c4c7e85138368">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>26. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001091">test_redirect_to_public_chat_tapping_tag_message</a>, id: 5675 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/785641a0359640058643d9c9b13c19e8?auth=fd13368ed08c3dfca989055863aa21e9">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>27. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001064">test_keycard_can_see_all_transactions_in_history</a>, id: 6291 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/97f6f0fc308242b7b6425efd1b683249?auth=fa66ebebd32cce828e75f2c7cc959077">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>28. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001062">test_keycard_send_two_transactions_one_after_another_in_dapp</a>, id: 6288 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2a782b55469749c2bfc7ddb346eab30c?auth=9086376701e95f69d90ce3d07fe7b65b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>29. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001110">test_push_notification_1_1_chat</a>, id: 6283 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/8907ab34ba314266a4922e19bdd216e1?auth=2be61a9a7dcc1b43044a8bf764088fe8">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/316b2c4be071405297b81a0a4f0cd13b?auth=46f6025603af2c189e33aa971e626073">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>30. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001053">test_can_use_purchased_stickers_on_recovered_account</a>, id: 5783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5ed3a373605445fd9404d603f390a3b5?auth=5b9177e4ae15de20abbfd1142cec13c2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>31. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001024">test_text_message_1_1_chat</a>, id: 5305 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/be971ccbf87f4c429d762f57b36dc0b3?auth=f18537b68288bc2f6b7956f15d50b85d">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/165a8ed0a2f44315b9c7144994af1921?auth=469610ac5a503e48cd1bafdf5ce34291">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>32. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001081">test_timestamp_in_chats</a>, id: 5385 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6e8c2c675f5147acaedade211628e8ef?auth=20c6a5b51063e58634ab1196cb88c1ab">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/e2ffbd8a389b4869aa982f618b9d271a?auth=c7f04df2d8628da41f64113eb7d637f1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>33. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001095">test_mobile_data_usage_popup_continue_syncing</a>, id: 5741 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/a28fcbcc637f459391834d5c5a676576?auth=617c83c5095c2430cae83c922a883221">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>34. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001090">test_insufficient_funds_wallet_positive_balance</a>, id: 5412 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/3d6db55576de43ec8de9517a612050d9?auth=bee1bb48b6fe5dec8e03f4b1afa11364">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>35. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001113">test_recover_account_from_new_user_seedphrase</a>, id: 6296 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/4c9603dcdacf45acb1fea882acaa283a?auth=020868564f6a850f8c23e5de24fbb0ec">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>36. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001026">test_request_stt_from_daap</a>, id: 5309 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/3adddfdf86df45b69093e8fc77030d0a?auth=a3d4fc02b4f23895bc9694cebbd63c2e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>37. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001082">test_public_chat_clear_history</a>, id: 5386 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/25196cc399df4bdabbc9b121ca5534ab?auth=13f50597d346db4cf5675e6e5b4ee254">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/69dd194dd6a6410a8529a29a23211e4f?auth=6f2680472013a52b463a0edbede944d0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>38. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001078">test_copy_contact_code_and_wallet_address</a>, id: 5375 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/60d5dbb773f04f2ba3a15205e5b87f38?auth=3bf8e82111361539f33b16f696445ff0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>39. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001068">test_send_audio_message_with_push_notification_check</a>, id: 6316 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/cac2d259d7594127a5f9298007b48858?auth=76f5d0a072a48c3d5354ce91594d12ad">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/7cbab989c9f940979232bf36eef7b42c?auth=77688bad3ddf00b6ac13559fd2b33eb5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>40. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001038">test_messaging_in_different_networks</a>, id: 5338 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/df6cd5e9700d479ab4e1aad3afb7cd58?auth=fcba1d17a850ef070cb61eae819fcf39">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/fc12e9c998f64ba894c04af68fe3a9ff?auth=2ca74da78dc32e34e6edb24c0da451bd">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>41. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001043">test_refresh_button_browsing_app_webview</a>, id: 5354 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/7a21a7f86dcc4e86b6582fc513415b2a?auth=3fbbb39806860f38bb6f02835d2311a5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>42. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001099">test_fetch_more_history_in_empty_chat</a>, id: 6205 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/92b08600ee014a64bfe3da4d03a66b8e?auth=d3f640b98099d3ce7c6bc787836884ee">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>43. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001075">test_collectible_from_wallet</a>, id: 5346 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/4c43346b1a484a8cb5be13c67e711095?auth=bcb30de5fdd3a3d883d68720d4ce0284">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>44. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001063">test_keycard_send_eth_from_wallet_to_address</a>, id: 6289 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/919efbb4e7b9405fb63d2c5d2f49df43?auth=561a1eec02bbe5b13cc9b6a301a552ac">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>45. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001028">test_login_with_new_account_logcat</a>, id: 5312 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1f14d184fc6d4446bf28c1228e8f6bd0?auth=c17c6c35a022a914330e2385a0451f39">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>46. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001103">test_mobile_data_usage_settings</a>, id: 6229 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b2546998bb0e45f99d5d602dd7cb9023?auth=d2675bf0cd849ff75c1e78d883937ec9">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>47. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001076">test_pass_phrase_validation</a>, id: 5363 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/edc3c5d1bd714472826b2dca882818b0?auth=5ab95f8f0063edbd36bb3ebd4bc7129c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>48. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001041">test_send_token_with_7_decimals</a>, id: 5350 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/837d1ca5409948a6be85908ae75b819f?auth=88a392fedca2d2f2a424e640a69e5efd">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>49. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001027">test_offline_is_shown_messaging_1_1_chat</a>, id: 5310 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/cedbd840b1e742df952e0c47f2d0b62b?auth=7e540ce7e840083aa44745fb6fad60fc">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/a1842fb7ac5046b385b3d8fd31c0340c?auth=4cf17e7e78340fa7123649aa947aa821">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>50. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001054">test_block_user_from_public_chat</a>, id: 5786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/bcd9c4a94f844ce199da7034a87b9644?auth=dcdc7fa1d06d51e3c4c0330eb33fc469">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/824fb2ed08f14c40bba94790a7304cc3?auth=081d2a8873310b4c7791d200130d5d41">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>51. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001084">test_long_press_delete_clear_all_dapps</a>, id: 5390 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6af3f4d9053d4bbabcb674a63c6ae961?auth=2f5619283cf2713912c84e6ba0d9c0a6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>52. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001056">test_ens_mentions_and_nickname_in_public_and_1_1_chats</a>, id: 6226 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1104b4d0c80e43e89539110c51d7e47f?auth=b210735acdead9598f0dfd51d26388e9">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/e06510d6a809469aba692a86c6617572?auth=e77e2098e87cfd3fc106bc477565992c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>53. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001036">test_send_emoji</a>, id: 5328 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/a5b1cd1327774749995a1a75f6708b50?auth=2f458831c43c1cadd9f452e646965f3e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>54. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001040">test_sign_message_from_daap</a>, id: 5342 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e83257368d22492ebf5abc5b54d082d3?auth=51ccbe6ec5500376a3dbaffdb422ed98">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>55. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001042">test_filters_from_daap</a>, id: 5353 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/05e30baf6ab948aea6a61e14cd06e214?auth=ceca972c30e9b3fc1c9c9a8f8929849b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>56. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001092">test_onboarding_screen_when_requesting_tokens_for_recovered_account</a>, id: 5677 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/bd4b431d58834aca9242fd84430820d0?auth=1766f39010e7a8b63d22470af813fd17">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>57. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001069">test_create_new_group_chat</a>, id: 3994 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6a82e8b3341d4c58952b60c9c6c1e6cd?auth=132647e8660d035ee220938d458c6673">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/6e6e43b0c06440c0aafb492b0fbc4af0?auth=5c47d2924d827251d8a589b29ac01789">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>58. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001033">test_long_press_to_delete_chat</a>, id: 5319 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/14ee590954904356b91e1645268afa1a?auth=009ac872d042838876cd06adb6d13428">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>59. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001077">test_send_and_open_links</a>, id: 5373 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/7ba7961fbeaa4700bacbfe1e26e3e696?auth=887f966038dd5551ac35a9fbdd0dbffc">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/abc16cfe21274a9989be7541da568568?auth=1afcf433426bf7926e90fea0c29e19d5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>60. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001087">test_open_public_chat_using_deep_link</a>, id: 5396 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0025e277b72f488db9c244056616404d?auth=927ee25d6a1a408e11a2f0d1cd45818e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>61. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001032">test_copy_and_paste_messages</a>, id: 5317 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ba37d2671fa54be8b3af03595aaeb908?auth=0532740774dac57f348abb3080843840">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>62. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001107">test_keycard_request_stt_from_daap</a>, id: 6249 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/3fc01d1f855b4a939a67aa30b2f32d35?auth=c9ca2b9d437b8432dda60a8940004196">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>63. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001045">test_unread_messages_counter_public_chat</a>, id: 5360 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/a9e1553d95744e6ea8a1b2a47e6ab492?auth=3580c45983093a59ff032a9d470489af">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/6f0ddbbe09a045e4a36344c774027550?auth=a2d7abacd37c7c99ed5f24bfc3faee12">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>64. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001094">test_dapps_permissions</a>, id: 5738 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/bd01edeb8e4b45de9184877b09ccb912?auth=c48c163c581953c043bcda3a045eb678">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>65. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001039">test_manage_assets</a>, id: 5341 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/cbf1756ae89b409785144c2a5fa6f506?auth=6796180799cc85f2908fca846f5c54c4">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>66. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001035">test_share_contact_code_and_wallet_address</a>, id: 5323 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d86ab70057814cb7961863f219a910cc?auth=080dad759f54296641ed8a71ec352ad3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>67. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001046">test_open_transaction_on_etherscan</a>, id: 5384 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0202fdd53fb244babd3d63ada9861502?auth=6e915644909630a6dd31784c1d6085b2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>68. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001088">test_request_public_key_status_test_daap</a>, id: 5397 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/808c99e324674a9a83270e3da3fea6ac?auth=91c7fa57a835ea8ed811db0096e24179">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>69. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001105">test_restore_account_from_mnemonic_to_keycard</a>, id: 6240 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/4ca50042723f47ad844477d241dee589?auth=4d73c0711440085d4489babe9f07c865">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>70. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001086">test_account_recovery_with_uppercase_recovery_phrase_logcat</a>, id: 5394 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/4a85407e2e82451d88a1ac04db8936b1?auth=4104722f98d89c6067cb22c74c0c3ca5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>71. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001080">test_user_can_see_all_own_assets_after_account_recovering</a>, id: 5381 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/4bdf049bf2804d148a6f4f8fd2072960?auth=89b001df19f9f732a1f71ffbb27bb895">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>72. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001031">test_add_to_contacts</a>, id: 5316 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/a0460ecf59524cefa1a63e0cd323783e?auth=77f3d7261c5c18b89e9b648b4293d1b1">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/d60f85be36684d40a1573c477c8c6d0b?auth=4863cca651881841c8f58b158e0d70aa">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>73. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001096">test_send_two_transactions_in_batch_in_dapp</a>, id: 5743 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/8fce3d94642848a6a7e2befd00b46157?auth=39ff06f3e984d426cea689ac89b27cf0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>74. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001111">test_keycard_send_two_transactions_in_batch_in_dapp</a>, id: 6287 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/91020173d05041138e855d5fef43c172?auth=46f423ccd5558dd38e0e2e699ded9ffe">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>75. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001112">test_keycard_fetching_balance_after_offline</a>, id: 6290 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0f9b329d23cd46e6ba383dfdc11d86eb?auth=de19bc58b7501a024bbd101eef6dd19e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>76. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001037">test_add_and_remove_mention_contact_with_nickname_from_public_chat</a>, id: 5332 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/dad9e7f9b0504de7b07935cf19847d24?auth=2cd3a8a82268144f762d5f1e73545bb8">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/e8a52b23e93740bc99983962a7173930?auth=626e72f9c2b7964f598fbcee87cf8d13">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>77. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001097">test_keycard_can_recover_keycard_account_card_pairing</a>, id: 5758 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/3158d084aea24f26b56640c879faad73?auth=ccb0b44991185712cf16d911714cd330">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>78. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001083">test_delete_chats_via_delete_button</a>, id: 5387 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5259b24da04f47c6ade791e857814d81?auth=f91e50a03c506c44b1e0ce2ce9b86fd6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>79. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001073">test_send_non_english_message_to_newly_added_contact</a>, id: 5315 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/89e61da11ac345148e2bc26badda4a4d?auth=27d44fc365b33cdf857959bb7139a6bc">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/defed68b075d4e6fa439dbe96244cc27?auth=cce3885d12dc51855df2cc945f024770">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>80. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001072">test_open_chat_by_pasting_public_key</a>, id: 5304 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/bba36d7c92674a908e3cbf04a4baaa05?auth=6e85f16e6f20c46ca541f33eaacb3c2c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>81. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001048">test_ens_username_recipient</a>, id: 5406 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2fa721415e5947e1b632dc083fb952b9?auth=f8dbbcf86f21cf2e4e93078550a75f1e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>82. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001051">test_send_two_transactions_one_after_another_in_dapp</a>, id: 5744 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/3f7b93122d2741bdb43f18d88c4c9495?auth=e3557ff85f6b4ed7fdc73b67ee6bb91b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>83. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001098">test_pair_devices_sync_one_to_one_contacts_nicknames_public_chat</a>, id: 5762 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6e70ecba8272470886cdca5bb0a61cee?auth=30140c90546a3b93629d609b95885d03">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/61a3bcfe0d6a494cba6d153c381009be?auth=337408850679d8cdb7653749bb837972">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>84. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001065">test_keycard_send_eth_in_1_1_chat</a>, id: 6293 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2cb778cd5cf1419d8c57ae8802251603?auth=b2938282b8f3f90498a84424cf7d136a">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/729dd2064e0241f4b646e45a3b6b6253?auth=91e5df3fcd247b14dd45841304c83693">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>85. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001066">test_image_in_one_to_one_send_save_reply_timeline</a>, id: 6305 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/04ea75fa546342edb90d07a4d970b9a9?auth=a2ccf7df08052678a61afa006050dc08">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/9b14b7768a58447e83742109903c17e3?auth=12444398da2a8250b7853436b8e67fb8">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>86. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001100">test_send_transaction_with_custom_token</a>, id: 6208 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0e732e0e447344e4aafdb90f53fe30c6?auth=cb966e2cac135814c56932f07e1798bf">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>87. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001061">test_send_eth_to_ens_in_chat</a>, id: 6279 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/9e520a3930964a8b8644e01b1d7009c5?auth=202b692cc0f3c407ce467f7b301c0be3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>88. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2001093">test_add_new_keycard_account_and_login</a>, id: 5689 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0efbdc6ef8784e689afb76c5a82ca206?auth=1d9edbfab65f80d35fcc82e0374efeb1">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

flexsurfer commented:
ok if we want to fix that lets fix
1) we need to set this flag when we receive this error , but not when cancel (because while popup is opened we accumulate popups because we still receiving errors ) 
2) we need to reset this flag when we select another mailserver (lets say you connected to custom with error, ignore error and then connect to another and receive error, we shouldn't ignore it here )

cammellos commented:
cool, thank you, will do that

cammellos committed:
Don't allow multiple pop ups to be shown

This commit makes sure that we don't display multiple pop ups when
mailserver connection has failed.
It also fixes the navigation to avoid being stuck on Profile tab.

Signed-off-by: Andrea Maria Piana <andrea.maria.piana@gmail.com>

cammellos commented:
@flexsurfer thanks, I have addressed the suggestions.

The last thing that would be left to do is how you suggested to reset the checks once a re-connection takes place, this should be easy to do but I will address in a different PR if that's ok.
ready to be reviewed again.
